src/pages/Results/Results.js

```diff
@@ -60,12 +70,42 @@ const Results = () => {
                                     <Checkbox name={item.name} checked={checkedItems[item.name]?.checked} onChange={event => handleChange(event, key, checkboxes[key].predicate)} />
                                     <DifficultyExplanation difficulty={item.name} />
                                 </div>
-                                ))
+                            ))
                             }
                         </div>
                     ))}
                 </div>
             </div>
+            <div className="results-page-section display-phone">
+                {SearchSelected ? <ResultsBody
+                    text={text}
+                    checkedItems={checkedItems}
+                /> :
+                    <div className="filters-container">
```

Comment:
I see what you mean about the phone and desktop version... At the very least, if we're duplicating this much code, then perhaps we should create a separate component for this div and everything inside it? It won't solve the issue at its core but at least the code won't be as redundant